redisgears_v8_plugin/src/v8_backend.rs

```diff
@@ -207,6 +276,25 @@ impl BackendCtxInterfaceInitialised for V8Backend {
                 let ctx = isolate_scope.new_context(None);
                 let ctx_scope = ctx.enter(&isolate_scope);
 
+                let globals = ctx_scope.get_globals();
+                let propeties = globals.get_own_property_names(&ctx_scope);
+                propeties.iter(&ctx_scope).try_for_each(|v| {
+                    let s = v.to_utf8().ok_or(GearsApiError::new("Failed converting global property name to string"))?;
+                    if !GLOBALS_ALLOW_LIST.contains(s.as_str()) {
```

Comment:
Currently if something is on the allow list it will be taken. We can verify there is no overlaps on runtime when the server starts. But I guess it is better to do it on compile time using a proc macro? WDYT?

Reply:
Example of the compile time error:

```bash
> cargo build
   Compiling redisgears_v8_plugin v0.1.0 (/home/meir/work/RedisGears-2.0/redisgears_v8_plugin)
   error: "WebAssembly" value appears on both, the allow list and the deny list.
   --> redisgears_v8_plugin/src/v8_backend.rs:34:79
```